native/src/components/Tile.tsx

```diff
@@ -35,11 +35,14 @@ class Tile extends React.Component<PropsType> {
 
   render(): ReactNode {
     const { tile, theme } = this.props
+    console.log(tile.thumbnail.toString())
     return (
       <TileContainer>
         <TouchableOpacity onPress={this.onTilePress}>
-          <ThumbnailContainer source={tile.thumbnail} />
-          <TileTitle theme={theme}>{tile.title}</TileTitle>
+          <View>
```

Comment:
Is this change actually necessary? Seems to work the same way if not wrapped in a `View`.